docs/openapi/components/schemas/presentations/TraceablePresentation.yml

```diff
@@ -2,7 +2,7 @@ $linkedData:
   term: TraceablePresentation
   '@id': https://w3id.org/traceability#TraceablePresentation
 title: Traceable Presentation
-description: A JSON-LD Presentation which establishes traceability linkage to a workflow instance and workflow type. 
+description: A JSON-LD Presentation which establishes traceability linkage to a workflow instance and workflow definition. The first Traceable Presentation establishes an instance identifier. The Workflow Instance identifies this particular workflow. Subsequent presentations indicates correlation by referencing the same instance identifier. The Workflow Definition indicates the intention of the presentation. 
```

Comment:
`The Workflow Definition indicates the intention of the presentation.`
Needs the same wording change as line 19 of `Workflow.yml`, above